### ISSUE TITLE
Bug: false unused reports with eslint 

### ISSUE THREAD

**Issue by Barkole** [depcheck/depcheck#552]
All the following dependencies are required, but marked as unused by depcheck.

Tested with https://gitlab.com/Lepardo/simplewebspider-nodejs/-/tree/51ee6269e3b9bd405c68b92a5691e44d3a196c12 (<- the ignores are active)

```txt
$ yarn depcheck
yarn run v1.22.4
$ depcheck
Unused devDependencies
* @typescript-eslint/eslint-plugin
* @typescript-eslint/parser
* eslint-config-prettier
* eslint-config-standard
* eslint-plugin-import
* eslint-plugin-node
* eslint-plugin-prettier
* eslint-plugin-promise
* eslint-plugin-standard
error Command failed with exit code 4294967295.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

$ yarn remove @typescript-eslint/eslint-plugin
yarn remove v1.22.4
[1/2] Removing module @typescript-eslint/eslint-plugin...
[2/2] Regenerating lockfile and installing missing dependencies...
success Uninstalled packages.
Done in 17.64s.

Mike@media MINGW64 /d/vscode/simplewebspider-nodejs (master)
$ yarn remove @typescript-eslint/parser
yarn remove v1.22.4
[1/2] Removing module @typescript-eslint/parser...
[2/2] Regenerating lockfile and installing missing dependencies...
success Uninstalled packages.
Done in 15.68s.

$ yarn lint
$ eslint --quiet --fix --report-unused-disable-directives src/**/*.{js,ts,tsx}

Oops! Something went wrong! :(

ESLint: 7.3.0

ESLint couldn't find the plugin "@typescript-eslint/eslint-plugin".

(The package "@typescript-eslint/eslint-plugin" was not found when loaded as a Node module from the directory "D:\vscode\simplewebspider-nodejs".)

$ yarn lint
$ eslint --quiet --fix --report-unused-disable-directives src/**/*.{js,ts,tsx}

Oops! Something went wrong! :(

ESLint: 7.3.0

Error: Failed to load parser '@typescript-eslint/parser' declared in '.eslintrc.cjs': Cannot find module '@typescript-eslint/parser'
Require stack:
- D:\vscode\simplewebspider-nodejs\.eslintrc.cjs
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.resolve (internal/modules/cjs/helpers.js:78:19)
    at Object.resolve (D:\vscode\simplewebspider-nodejs\node_modules\eslint\lib\shared\relative-module-resolver.js:28:50)
    at ConfigArrayFactory._loadParser (D:\vscode\simplewebspider-nodejs\node_modules\eslint\lib\cli-engine\config-array-factory.js:931:45)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (D:\vscode\simplewebspider-nodejs\node_modules\eslint\lib\cli-engine\config-array-factory.js:733:43)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigData (D:\vscode\simplewebspider-nodejs\node_modules\eslint\lib\cli-engine\config-array-factory.js:647:20)
    at _normalizeObjectConfigData.next (<anonymous>)
    at ConfigArrayFactory.loadInDirectory (D:\vscode\simplewebspider-nodejs\node_modules\eslint\lib\cli-engine\config-array-factory.js:495:28)
    at CascadingConfigArrayFactory._loadConfigInAncestors (D:\vscode\simplewebspider-nodejs\node_modules\eslint\lib\cli-engine\cascading-config-array-factory.js:355:46)

$ yarn remove eslint-config-prettier
yarn remove v1.22.4
[1/2] Removing module eslint-config-prettier...
[2/2] Regenerating lockfile and installing missing dependencies...
success Uninstalled packages.
Done in 14.69s.

$ yarn lint
$ eslint --quiet --fix --report-unused-disable-directives src/**/*.{js,ts,tsx}

Oops! Something went wrong! :(

ESLint: 7.3.0

ESLint couldn't find the config "prettier/@typescript-eslint" to extend from. Please check that the name of the config is correct.

The config "prettier/@typescript-eslint" was referenced from the config file in "D:\vscode\simplewebspider-nodejs\.eslintrc.cjs".

If you still have problems, please stop by https://eslint.org/chat to chat with the team.
```

**Comment by Barkole** [comment]
Also `eslint-plugin-no-only-tests` is reported, but used by eslint

**Comment by znarf** [comment]
Not reproducing with latest version. `No depcheck issue`
